### ISSUE TITLE
Custom TabStyles

### ISSUE THREAD

**Issue by ianlaidlaw** [iRoachie/react-native-material-tabs#32]
Sorry for another request - Component has been fantastic otherwise.

I noticed that 12px horizontal padding is applied to all tabs. I would like to override this, or maybe all other tabStyles. Is this something you have considered?

**Comment by iRoachie** [comment]
Yes! I did think about doing it, do you think you could send a PR to implement this?

**Comment by ianlaidlaw** [comment]
Sure, I will work on this when I have time!

**Comment by kirill-konshin** [comment]
Also there are cases when tab width can be based on content, not justified. Current implementation does not allow that, it sets width by force (but even this behavior can be achieved by pure CSS with no calculation).

**Comment by vorasudh** [comment]
Any progress on this enhancement?
